domain/src/main/java/no/difi/meldingsutveksling/core/EDUCoreMarker.java

```diff
@@ -0,0 +1,26 @@
+package no.difi.meldingsutveksling.core;
+
+import net.logstash.logback.marker.LogstashMarker;
+import no.difi.meldingsutveksling.logging.MarkerFactory;
+
+public class EDUCoreMarker {
+
+    /**
+     * Creates LogstashMarker with id from the request that will appear in the logs when used.
+     *
+     * @param message request that contains journal post id, receiver party number, sender party number and conversation id
+     * @return LogstashMarker
+     */
+    public static LogstashMarker markerFrom(EDUCore message) {
```

Comment:
Dette blir utbedret når vi får laget optimaliserte typer for henholdsvis EDU og AppReceipts.